common/src/models/request/tokenRequest.ts

```diff
@@ -0,0 +1,90 @@
+import { TwoFactorProviderType } from "../../enums/twoFactorProviderType";
+import { Utils } from "../../misc/utils";
+
+import { CaptchaProtectedRequest } from "./captchaProtectedRequest";
+import { DeviceRequest } from "./deviceRequest";
+
+export class TokenRequest implements CaptchaProtectedRequest {
```

Comment:
Curious about this added file?

Reply:
Probably a file that was removed in a PR and got accidently re-added.